content/english/security/1- Meeting Alice, Bob and Eve/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Meeting Alice, Bob and Eve"
+description: ""
+draft: true
```

Comment:
can you change this to false to see how it looks here? https://workshops-dev.nuevofoundation.org/

Reply:
modified